### ISSUE TITLE
Label filtering in new issue page

### ISSUE THREAD

**Issue by cypherhoodlum** [NostrGit/NostrGit#164]
#162 must be merged before starting with this issue.

Users should be able to filter the label listing when adding them to a new issue on [entity]/[repo]/issues/new.

The input field exists, but the actual filtering is not implemented yet.
![Screenshot from 2023-03-11 20-41-48](https://user-images.githubusercontent.com/82510823/224506015-8c2e630a-03a7-4d67-b2f4-2ff9a00b459a.png)

**Comment by gitstart** [comment]
We are working on this
